apps/avifenc.c

```diff
@@ -396,18 +399,7 @@ int main(int argc, char * argv[])
     int keyframeInterval = 0;
     avifBool cicpExplicitlySet = AVIF_FALSE;
     avifBool premultiplyAlpha = AVIF_FALSE;
-
-    // The timescale specifies how many units of time are in a second, and the duration is how many
-    // of those timescales this frame should be displayed for. For example, if they match, it signals
-    // 1 fps. If duration is 1 and timescale is 30, it signals 30 fps.
-    //
-    // These are set to 0 as sentinel values on initialization, and (if unchanged) are set to actual
-    // defaults later (search for "Set timing defaults"). This is done so that if the commandline
-    // doesn't set either of these values and the first source image provides a framerate, avifenc
-    // can use it instead of a 1 fps default.
-    uint64_t duration = 0;
-    uint64_t timescale = 0;
-
+    avifAppSourceTiming sourceTiming = { 0, 0 }; // See declaration of avifAppSourceTiming for documentation
```

Comment:
Nit: This variable name is a little confusing because it's not clear what "source" means. Note that this variable stores the values from the command line.

Reply:
I only named it that to make it clear it was an `avifAppSourceTiming` instead of an `avifImageTiming`, but you're right that this specific variable could have a better name. Suggestions? I could see it being `outputTiming`, perhaps?

Reply:
`timingDefaults` seems even more appropriate. Thoughts?

Reply:
I went with `outputTiming`. I'll push this now.

Reply:
"outputTiming" or "animationTiming" sounds good.